packages/std/src/traits.rs

```diff
@@ -162,17 +182,21 @@ impl<'a> QuerierWrapper<'a> {
         }
     }
 
-    pub fn query_balance<U: Into<HumanAddr>>(&self, address: U, denom: &str) -> StdResult<Coin> {
+    pub fn query_balance<U: Into<String>, V: Into<String>>(
+        &self,
+        address: U,
```

Comment:
`&str` implements `Into<String>` via cloning. So `query_balance(addr.as_ref())` because it gives you a `&str`.

`query_balance(&addr)` probably does not yet work.

Reply:
Implemented in 82def5e1beb1dafbede8cc89d7bbd50d6dce7a94